packages/block-library/src/post-title/edit.js

```diff
@@ -73,3 +69,20 @@ export default function PostTitleEdit( {
 		</>
 	);
 }
+
+export default function PostTitleEdit( {
+	attributes,
+	setAttributes,
+	context,
+} ) {
+	if ( ! context.postType || context.postType === 'wp_template' ) {
```

Comment:
I wonder if we should switch this to the approach used for the post content block to make it more flexible in other post context :thinking:

Reply:
I actually think using PostTitle inside PostContent of the same context is fine. Why shouldn't it be possible?

Reply:
Ah yes, looking back this is the 'weird spot' i wasn't sure about.  Honestly we should probably just remove the changes here as they don't induce any crashing.

> I actually think using PostTitle inside PostContent of the same context is fine. Why shouldn't it be possible?

It should be possible!  However, there are some oddities.

Say we are using the template post type editor to create a template that uses the post title block.  The intention would be that it displays the post title for whatever context is given to that page in the site editor, but editing that in the template post type editor it would be showing the name of the template instead.

Reply:
I agree that the placeholder would be useful here, but yeah, maybe there is a better approach for accomplishing that.

Reply:
I removed the changes to Post Title.  We can revisit this later if it is an issue, but for this PR we should concentrate on fixing the editor crashing via the post content block.

Reply:
sounds good!